bricksrc/equipment.py

```diff
@@ -7,14 +7,17 @@
 equipment_subclasses = {
     "HVAC": {
         OWL.equivalentClass: BRICK["Heating_Ventilation_Air_Conditioning_System"],
+        "tags": [TAG.Heating, TAG.Ventilation, TAG.Air, TAG.Conditioning, TAG.System],
```

Comment:
Do we want to have both the `HVAC` and `Heating`, `Ventilation`, etc tags at the same time? They are equivalent classes. I like having just the `HVAC` tag (probably `HVAC` + `System` belong together?) but I could be convinced otherwise